x-pack/docs/en/security/authorization/privileges.asciidoc

```diff
@@ -171,10 +171,17 @@ Privilege to index documents. Also grants access to the update mapping action.
 However, it does not enable a user to update existing documents.
 +
 --
-NOTE: When indexing documents with an external `_id` either via the index API or
-the bulk API, the request must use `op_type` as `create`. If `_id`s are
-generated automatically, the authorization happens as if the `op_type` is set to
-`create`.
+[NOTE]
+====
+This privilege relies on the `op_type` of indexing requests (<<docs-index_>> and
```

Comment:
> This privilege relies on....

It seems to me that the implication here is that when you have only create_doc privilege, you cannot use the index or bulk APIs successfully unless you use one of the three options below.  If that's true, I don't think we're making that dependency clear enough.  I'd suggest changing "the op_type can be set to create..." to something like "...you must set the op_type to create..."

Reply:
Perhaps something more like _you must ensure that the op_type is set to create_ ?